docs/ucai/index.md

```diff
@@ -0,0 +1,253 @@
+# Welcome to the Unity Catalog AI documentation
+
+## Defining a Client
+
+To interact with Unity Catalog (UC) functions, you must define and initialize a client that manages communication between your application and the UC service. The `DatabricksFunctionClient` provided in the Unity Catalog AI Core Library is designed to facilitate this interaction seamlessly.
+
+### Client initialization
+
+Here is how to initialize a client for connecting to a managed Databricks Unity Catalog:
+
+``` python
+from ucai.core.databricks import DatabricksFunctionClient
+
+client = DatabricksFunctionClient(warehouse_id="YOUR_WAREHOUSE_ID")
+
+CATALOG = "my_catalog"
+SCHEMA = "my_schema"
+```
+
+If you decide to run operations using serverless runtime on Databricks, you do not need to define a `warehouse_id` when creating the client.
+
+#### Gotchas and Best Practices for the Client
+
+- **Warehouse ID Accuracy**
```

Comment:
Do you think table is better here for displaying?

Reply:
yep!